what-the-stack/src/main/java/com/haroldadmin/whatthestack/ui/components/ScrollableText.kt

```diff
@@ -0,0 +1,27 @@
+package com.haroldadmin.whatthestack.ui.components
+
+import androidx.compose.foundation.horizontalScroll
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontFamily
+
+@Composable
+internal fun ScrollableText(
```

Comment:
This is only used once so it doesn't make a lot of sense to separate it out, unless you have plans to make use of it in the future.